src/App.tsx

```diff
@@ -1,7 +1,33 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends Component {
+  state = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', (evnt: KeyboardEvent) => {
+      this.setState({ pressedKey: evnt.key });
+    });
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', (evnt: KeyboardEvent) => {
```

Comment:
You need to remove the event listener with the same function reference used in `componentDidMount`. Create a class method named `handleKeyUp` and use it as the event listener in both `componentDidMount` and `componentWillUnmount`.